src/client/sandbox/event/listeners.js

```diff
@@ -296,4 +296,18 @@ export default class Listeners extends EventEmitter {
             eventListeningInfo.outerHandlersWrapper = wrapper;
         }
     }
+
+    getEventListeners (el, event) {
+        const eventCtx = this.listeningCtx.getEventCtx(el, event);
+
+        if (!eventCtx)
+            return null;
```

Comment:
I propose to return an empty array in this case.
It allows to simplify code that will be use it.